src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerTest.cs

```diff
@@ -1221,7 +1221,7 @@ public void ConsoleLoggerOptions_TimeStampFormat_IsReloaded()
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
         public void ConsoleLoggerOptions_TimeStampFormat_IsReadFromLoggingConfiguration()
         {
-            var configuration = new ConfigurationBuilder().AddInMemoryCollection(new[] { new KeyValuePair<string, string>("Console:TimeStampFormat", "yyyyMMddHHmmss") }).Build();
+            var configuration = new ConfigurationBuilder().AddInMemoryCollection(new[] { new KeyValuePair<string, string>("Console:TimestampFormat", "yyyyMMddHHmmss") }).Build();
```

Comment:
I don't think we should be moving forward with this change given this regression. Configuration is case-insensitive by design. Breaking this means anyone who is using environment variables with all caps, or has different casing in their appsettings.json file, will be broken.

I think we should wait for the underlying case-insensitive fix so we don't regress anyone in the meantime. To my knowledge, there isn't a need to merge this change before getting the casing fixed.

Reply:
That's fair. I'll fix the case-insensitive issue first.

Reply:
I agree with @eerhardt here after hearing his perspective.  Unlike the source-generator where its new and opt-in here we are changing the behavior of existing API for everyone.  That has the potential to break tests and benchmarks if we merged it into a nightly build.

Reply:
Fixed in https://github.com/dotnet/runtime/pull/88338.